app/css/pages/community-solution-show.css

```diff
@@ -139,7 +139,7 @@
                             @apply mb-4 flex;
                             @apply text-16 leading-150 text-textColor2;
                             & strong {
-                                @apply font-semibold;
+                                @apply font-semibold flex gap-4;
```

Comment:
You could also maybe just do this as inline or inline flex and let the browser handle the spacing. But I'm ok with this too :)

Reply:
yeah, that is nicer!